packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/upgrade-guides.5/version-5-upgrade-guide.5.tool.md

```diff
@@ -0,0 +1,71 @@
+# Terra Functional Testing - Version 5 Upgrade Guide
+
+The upgrade from v4 to v5 removes `@wdio/async`
+
+## Breaking Changes
+
+In order to prepare for upgrading the Terra ecosystem to be compatible with Node +18, `@wdio/sync` is being removed as a dependency from `@cerner/terra-functional-testing`. As `fibers` and `@wdio/sync` are being deprecated from Node +16, this move is necessary to bring the Terra ecosystem with the latest Node version. Due to `@wdio/sync` being removed, all tests are now run asynchronously and tests will need to be re-written in order to ensure they run asynchronously. A short introduction to changing your tests to run asynchronously can be seen here: [From Sync to Async](https://webdriver.io/docs/async-migration/)
```

Comment:
I think the main focus of this paragraph should be the deprecation of the synchronous api instead of our future plans. We can probably say something like:

```suggestion
The synchronous WebdriverIO API (`@wdio/sync`) has been [deprecated](https://webdriver.io/blog/2021/07/28/sync-api-deprecation/) and is no longer supported. Therefore, terra-functional-testing is switching over to use the asynchronous webdriver API. This will also allow for Terra to utilize future versions of Webdriver and Node which do not support `@wdio/sync`. 

Due to `@wdio/sync` being removed, all tests are now run asynchronously and will need to be re-written. A short introduction to changing your tests to run asynchronously can be seen here: [From Sync to Async](https://webdriver.io/docs/async-migration/)
```